test-alerts.cpp

```diff
@@ -3,6 +3,92 @@
 #include "test/catch.hpp"
 #include "typewise-alert.h"
 
-TEST_CASE("infers the breach according to limits") {
-  REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+TEST_CASE("Passive Cooling system with Normal temp and O/P fed to controller") {
+	
+	Alert_Status_s Alert_Status_Output = {NO_PRINT_DONE,DEFAULT_BREACH};
+	BatteryCharacter batteryCharacter = {PASSIVE_COOLING,"Brand_TCQ"};
+	
+	Alert_Status_Output = checkAndAlert(TO_CONTROLLER, batteryCharacter, 20);
```

Comment:
The REQUIRE statements convey the functionality of checkAndAlert as a conversion from 3 parameters to one return value. If you had to state its function as a data-conversion, what would you state?